src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -154,6 +154,13 @@
 --    added: GRANT SELECT,UPDATE ON  nod_seq TO darts_user;
 --    updated annotation_document - all columns NOT NULL
 --    updated transcription_document - all columns NOT NULL
+--v53 amend transaction.hearing_date from TIEMSTAMP to DATE, to be consistent with hearing.hearing_date

```

Comment:
Quite a few typos in this comment

Reply:
transaction replaced by transcription and TIEMSTAMP replace by TIMESTAMP.  Does 2 equate to a few, but two too many for a single line. correction will be in v55